tests/data/view_modal_009.json

```diff
@@ -35,7 +35,7 @@
   "callback_id": "view_4",
   "external_id": "some-unique-id",
   "state": {
-    "values": []
+    "values": {}
```

Comment:
This has been a potential issue of test data. The `values` usually cannot be an array.